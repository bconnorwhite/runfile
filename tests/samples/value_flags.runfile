# ----------
# Value Flags
# ----------

# Value flags with type hints
build --output=<file> --target=<triple>:
  cargo build --output=$output --target=$target

# Mixed boolean and value flags
deploy --env=<name> --dry-run:
  echo "Deploying to $ENV environment"
  if [ "$DRY_RUN" = "true" ]; then
    echo "Dry run mode"
  else
    echo "Live deployment"
  fi

# Short and long value flags
test -o, --output=<file> -t, --timeout=<seconds>:
  cargo test --output=$output --timeout=$timeout
