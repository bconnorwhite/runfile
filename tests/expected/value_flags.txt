Value Flags
  build             # Value flags with type hints
    --output
    --target
  deploy            # Mixed boolean and value flags
    --env
    --dry-run
  test              # Short and long value flags
    -o, --output
    -t, --timeout


